### PR TITLE
Add 92-disc-v5-research

### PR DESCRIPTION
**Pull request by oskarth**
cc @adambabik

### CODE REVIEW

File: 92-eth-discovery.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+## Preamble
+
+    Idea: #92-disc-v5-research
```
Comment (oskarth) [inline 92-eth-discovery.md]: Should we rename this to #92-eth-discovery so it is the same as Slack channel (IIRC)?

File: 92-eth-discovery.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    Title: Research Ethereum discovery protocol
+    Status: Draft
```
Comment (oskarth) [inline 92-eth-discovery.md]: This is In Progress I think?

File: 92-eth-discovery.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    Created: 2018-03-12
+
+## Summary
+Discovery protocol is essential for long-term scaling of the Status app. It will improve security and reliability of the app and prepare it for the beta launch. 
+
+## Swarm Participants
+- Lead Contributor: @adambabik
+- Testing & Evaluation: TBD
```
Comment (oskarth) [inline 92-eth-discovery.md]: We need one here. Who is best suited for this type of technical evaluation you think?

File: 92-eth-discovery.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+- [ ] Can we turn on and off discovery protocol when the node is running?
+- [ ] Can Discovery V5 protocol carry more information than protocol name and version? How can we distinguish Light Whisper nodes or Whisper nodes with MailServer capability? Notice that revealing this information (like Light Whisper node) may increase a chance to reveal the peer identity. The fact that we don’t want mobile phones to be each other peers raises this question.
+- [ ] How to make sure that connecting to the cluster will be fast? This is more a cluster question, i.e. how to autoscale cluster so that it can quickly connect new peers.
+- [ ] How to make sure that there are always at least N peers with given protocol support. For instance, we always want to have at least some Whisper and some LES peers connected. Also, we want to always have at least one Whisper peer with MailServer support.
+
+### Requirements & Dependencies
+The discovery protocol should be testable locally with Docker, but for some more extensive tests, we may utilize a public cluster.
+
+### Minimum Viable Product
+Goal Date: TBD
```
Comment (oskarth) [inline 92-eth-discovery.md]: IMO MVP should be what we can get done before next town hall, so what can we do by April 6?

File: 92-eth-discovery.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+Description: 
+1. All questions described in Product Description are researched and answered,
+1. It's possible to start statusd and Status App with DiscV5 instead of static nodes (switch between them with a flag would be great),
+1. There is an automated way to run performance and resources consumption tests within an isolated environment.
+
+## Success Metrics
```
Comment (oskarth) [inline 92-eth-discovery.md]: Exit criteria section would be useful

File: 92-eth-discovery.md (hunk 1 above)
Comment (adambabik) [reply 92-eth-discovery.md]: On Slack it's `#92-disc-v5-research`.

File: 92-eth-discovery.md (hunk 3 above)
Comment (adambabik) [reply 92-eth-discovery.md]: That's a good question. No one has really talked to us about this idea yet.

File: 92-eth-discovery.md (hunk 4 above)
Comment (adambabik) [reply 92-eth-discovery.md]: We can definitely get it working locally in docker.

**Comment by adambabik** [comment]
@oskarth can we commit to this PR? What's the flow?

File: 92-eth-discovery.md (hunk 1 above)
Comment (oskarth) [reply 92-eth-discovery.md]: Oh right, sorry. Should probably rename this file then

**Comment by oskarth** [comment]
@adambabik Absolutely! It's yours for the taking (or if you prefer to create a new PR). I just added it to seed discussion. Same with https://github.com/status-im/ideas/pull/103

**Pull request merged**